src/Features/Core/Portable/Snippets/SnippetProviders/AbstractIfSnippetProvider.cs

```diff
@@ -24,7 +24,9 @@ internal abstract class AbstractIfSnippetProvider : AbstractSnippetProvider
     {
         public override string SnippetIdentifier => "if";
 
-        public override string SnippetDisplayName => FeaturesResources.Insert_an_if_statement;
+        public override string SnippetDescription => FeaturesResources.if_statement;
```

Comment:
fwiw, I think it's OK to use "Insert an if-statement" as inline description.

Reply:
I changed it so it would align with what VS Code says. There is an open question on whether or not the snippets will be discoverable/users will know what they do, so that extra description may help.

@mikadumont  what do you think?